sources/packages/web/src/router/AESTRoutes.ts

```diff
@@ -46,7 +46,7 @@ import AssessmentsSummary from "@/views/aest/student/applicationDetails/Assessme
 import StudentAppealRequestsApproval from "@/views/aest/student/applicationDetails/StudentAppealRequestsApproval.vue";
 import NoticeOfAssessment from "@/views/aest/student/applicationDetails/NoticeOfAssessment.vue";
 import ApplicationExceptionsApproval from "@/views/aest/student/applicationDetails/ApplicationExceptionsApproval.vue";
-import ViewScholasticStanding from "@/views/aest/student/ViewScholasticStanding.vue";
+import ViewScholasticStanding from "@/views/student/ViewScholasticStanding.vue";
```

Comment:
Each view page should have its own component, and then each page will share the same component.
For instance, student view pages use the `tudent-page-container` while Ministry pages use the `full-page-container`.

Reply:
Please review the comment where it states
> For instance, student view pages use the student-page-container while Ministry pages use the full-page-container.

<img width="552" height="473" alt="image" src="https://github.com/user-attachments/assets/20cf8906-d558-45f0-9364-262bfd657fe5" />